src/e3/env.py

```diff
@@ -279,7 +299,12 @@ def get_platform(
                     # Linux machine should not change the OS version
                     if split_value[1] is None:
                         split_value[1] = saved_build.os.version
-                return Platform.get(*split_value)  # type: ignore
+                return Platform.get(
+                    platform_name=split_value[0],
+                    version=split_value[1],
+                    machine=split_value[2],
```

Comment:
No minor version possible here ?

Reply:
Not in the current state at least, that would require another change (not necessarily very complex to have that said, but it wasn't in what was asked of me, and it would add a field to --target/--build/--host).

Note that my current change still allows you to get version_minor if the OS version in the parameter string uses a format like: MAJOR.MINOR[.MORE_SUB_VERSION]*